stdlib/LinearAlgebra/src/qr.jl

```diff
@@ -318,7 +318,9 @@ Iterating the decomposition produces the components `Q`, `R`, and if extant `p`.
 
 The following functions are available for the `QR` objects: [`inv`](@ref), [`size`](@ref),
 and [`\\`](@ref). When `A` is rectangular, `\\` will return a least squares
-solution and if the solution is not unique, the one with smallest norm is returned.
+solution and if the solution is not unique, the one with smallest norm is returned. When
+`A` is not full rank, factorization with (column) pivoting is required to obtain a finite
```

Comment:
I'm not sure about the "finite norm solution" part here. I'd expect it to be a "unique solution" or "minimum norm solution".

Reply:
You are right. After I submitted it, I also thought it was badly formulated. I will change it when I get home.